cmd/opentelemetry/app/receiver/kafkareceiver/kafka_receiver.go

```diff
@@ -145,6 +145,14 @@ func MustOtelEncodingForJaegerEncoding(jaegerEncoding string) string {
 		return "jaeger_proto"
 	case kafka.EncodingJSON:
 		return "jaeger_json"
+	case kafka.EncodingOTLPProto:
```

Comment:
This same method is used for the exporter which doesn't support some of these encodings.